website_sale_product_detail_attribute_image/models/product_attribute.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2019 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models, tools
+
+
+class ProductAttribute(models.Model):
+    _inherit = 'product.attribute'
+
+    website_product_detail_image = fields.Binary(
+        string='Website detail image',
+        attachment=True,
+        help='Small-sized image of the attribute for shop online product '
+             'detail. It is automatically resized as a 64x64 px image, with '
+             'aspect ratio preserved.',
+    )
+    website_product_detail_display = fields.Boolean(
```

Comment:
Being homogeneous with other namings, I would call this `website_product_detail_image_published`